1-org/modules/centralized-logging/main.tf

```diff
@@ -0,0 +1,204 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  value_first_resource  = values(var.resources)[0]
+  logbucket_sink_member = { for k, v in var.resources : k => v if k != var.logging_project_key }
+
+  all_sinks = merge(
+    var.bigquery_options == null ? {} : {
+      for k, v in var.resources : "${v}_bgq" => var.bigquery_options
+    },
+    var.logbucket_options == null ? {} : {
+      for k, v in var.resources : "${v}_lbk" => var.logbucket_options
+    },
+    var.pubsub_options == null ? {} : {
+      for k, v in var.resources : "${v}_pub" => var.pubsub_options
+    },
+    var.storage_options == null ? {} : {
+      for k, v in var.resources : "${v}_sto" => var.storage_options
```

Comment:
nit: Wouldn't it be simpler to just make this `"${v}_${local.storage_type }" => { options = var.storage_options, type = local.storage_type }` so below we dont have to do the subst approach?

Reply:
Done